### PR TITLE
cross app communication

### PR DESCRIPTION
**Pull request by saurabhsood91**
To enable cross app communication

### CODE REVIEW

File: sandstone/client/sandstone/components/updateservice/updateservice.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+angular.module('sandstone.updateservice', [])
+.service('UpdateService', ['$rootScope', function($rootScope){
+    var ws = null;
+    var setUpWebsocket = function() {
+        // TODO find better way to create WS address
+        var websocketAddress = "ws://" + window.location.hostname + ":" + window.location.port + "/messages";
```
Comment (zebulasampedro) [inline sandstone/client/sandstone/components/updateservice/updateservice.js]: Websocket protocol should be one of `ws` or `wss` depending on whether or not SSL is being used.

File: sandstone/apps/filebrowser/static/filebrowser.controller.js [hunk 2]
```diff
@@ -72,6 +72,16 @@ angular.module('sandstone.filebrowser')
     self.isCopied = true;
   };
 
+  self.openFileInEditor = function() {
+      var message = {
+          key: 'editor',
```
Comment (zebulasampedro) [inline sandstone/apps/filebrowser/static/filebrowser.controller.js]: @saurabhsood91 Let's namespace events with the ng app name. For example, `editor:openFile`.

File: sandstone/urls.py [hunk 3]
```diff
@@ -2,8 +2,8 @@
 from sandstone.lib.handlers.logout import LogoutHandler
 from sandstone.lib.handlers.main import MainHandler
 from sandstone.lib.handlers.state import StateHandler
+from sandstone.lib.handlers.update_handler import UpdateHandler
```
Comment (zebulasampedro) [inline sandstone/urls.py]: @saurabhsood91 For consistency with the rest of the handlers, rename the file from `update_handler` to `update`.

Comment (saurabhsood91) [inline sandstone/urls.py]: cool. will do that

**Pull request merged**